website/docs/intro/install/standalone-install.sh

```diff
@@ -0,0 +1,14 @@
+# Download the installer script:
+curl --proto '=https' --tlsv1.2 -fsSL https://get.opentofu.org/install-opentofu.sh -o install-opentofu.sh
+# Alternatively: wget --secure-protocol=TLSv1_2 --https-only https://get.opentofu.org/install-opentofu.sh -o install-opentofu.sh
+
+# Grand execution permissions:
+chmod +x install-opentofu.sh
+
+# Please inspect the downloaded script at this point.
+
+# Run the installer:
+./install-opentofu.sh --install-method standalone
```

Comment:
When I get to this step I hit the following
```
❯ ./install-opentofu.sh --install-method standalone
OpenTofu Installer

Cosign is not installed on your system, which is required to verify package integrity.
If you have cosign installed, please pass the --cosign-path option. Alternatively, you can disable integrity verification by passing --skip-verify (not recommended).
```

I'm using a pretty new install of Ubuntu 22.04.3 LTS. 

Running with skip verify does work though
```
❯ ./install-opentofu.sh --install-method standalone --skip-verify

OpenTofu Installer

Skipping integrity verification. This is not recommended.
Installing OpenTofu using the standalone installation method...
Determining latest OpenTofu version...
Downloading OpenTofu version 1.6.0-rc1...
Performing checksum verification...
Checksum for tofu_1.6.0-rc1_linux_amd64.zip is c810fa64d7147df4272681360ef5bc9a366e574bfdabd6b88b18e2c36c3b1aee, as expected.
Unpacking OpenTofu...
Archive:  /tmp/tmp.8ItiUU3lGe/tofu_1.6.0-rc1_linux_amd64.zip
  inflating: /tmp/tmp.4foOOeTi8Y/CHANGELOG.md
  inflating: /tmp/tmp.4foOOeTi8Y/LICENSE
  inflating: /tmp/tmp.4foOOeTi8Y/README.md
  inflating: /tmp/tmp.4foOOeTi8Y/tofu
Moving OpenTofu installation to /opt/opentofu...
[sudo] password for jjshanks:
Creating tofu symlink at /usr/local/bin/tofu...
Checking if OpenTofu is installed correctly...
OpenTofu v1.6.0-rc1
on linux_amd64
Installation complete.
```

Reply:
Thank you for testing the installation and pointing this out. This is intended behavior as we do not want to encourage users to install without verifying their download, but currently we only support cosign. However, tomorrow we are adding GPG verification support for the stable release too.

Was the explanation in the error message clear enough or can we do better here? We would really like people to have a good experience with this script.

Reply:
I think the error message is good but I'm not sure the experience is the best. Having some kind of notice that cosign is a requirement towards the top of the page instead of the bottom would be better I think. 

I actually missed the skip-verify part of the message and found it looking through the script. If possible moving the `Alternatively` sentence to the next line would be better for visibility.

Reply:
@cube2222 @cam72cam @Yantrio FYI, regarding the signing conversation we had and a possible use for code signing certificates.

Reply:
@jjshanks I've added an extra warning and the change in opentofu/opentofu.org#8 adds a much better explanation along with GPG support.